mitodl/ocw-www
# Add file thumbnail for resource search

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds a missing thumbnail image

#### How should this be manually tested?
Run `npm start`
Go to http://localhost:3000/images/file_thumbnail.png, you should see this:
![file_thumbnail](https://user-images.githubusercontent.com/187676/113051435-f30e4f80-9173-11eb-928b-f2a485087695.png)
